README.md

```diff
@@ -2,20 +2,49 @@
 This section describes tools that help you with coding. We provide scripts that allows to
 clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
 
+## Prerequisites on Windows
+Install and configure [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
+- Java 8 on WSL
+- Gradle 5+
+- Maven
+
 ## Clone all repositories
 Check a `development/pull-all.sh` script to clone all Knot.x repositories. Please check `-h` option
 for help. Execute this script from `knotx-aggregator/development` directory, executing it from another 
-directory will fail.
+directory will fail. Please check `-r` option for providing custom directory where Knot.x repositories 
```

Comment:
The `-r` parameter specifies a directory all Knot.x repositories are cloned, such as `sh pull-all.sh -r ../../knotx`. Please make sure that the provided directory exists.

Reply:
In this case the pointed directory exists for sure ;)

Reply:
fixed